runtime/onert/core/src/exec/train/TrainableExecutor.h

```diff
@@ -54,11 +54,14 @@ class TrainableExecutor : public IExecutor
 public:
   const ir::Graph &graph() const final { return _trainable_graph.graph(); }
 
-  void execute(const IODescription &desc) final;
+  void execute(const IODescription &desc) override { forward(desc, false); };
 
   void execute(const std::vector<backend::IPortableTensor *> &inputs,
                const std::vector<backend::IPortableTensor *> &outputs) override;
 
+  void forward(const IODescription &desc, bool training);
+  void backward(const IODescription &desc);
+
```

Comment:
@jyoungyun Do we need expose `forward` and `backward` as public?

Reply:
Yes, this functions will be called from nnfw internal api.